src/sctools/bam.py

```diff
@@ -464,7 +493,26 @@ def __repr__(self) -> str:
         return 'query_name'
 
 
-class CellMoleculeGeneQueryNameSortOrder(AlignmentSortOrder):
+class TagsQueryNameSortOrder(AlignmentSortOrder):
```

Comment:
Please update the docstring at the top of the file: https://github.com/HumanCellAtlas/sctools/blob/c2c2540801cf12d2f2fba72b2dabc6e82552a3d9/src/sctools/bam.py#L27